### ISSUE TITLE
[Suggestion]: Image Recognition to Initialize Farming Planner

### ISSUE THREAD

**Issue by jonathanlo411** [penguin-statistics/frontend-v2#516]
### 功能或改进建议的相关页面链接 // URL of the page that the suggestion is related to

https://penguin-stats.io/planner

## What is the feature or improvement related to?
1. Farming Strategy Planner

## How would you describe the improvement you would like to see?

Users would be able to upload screenshots from their depot and initialize the quantity of materials into the Framing Strategy Planner. This would greatly cut down on time for those first using the planner feature.

Sample image upload:
![](https://cdn.discordapp.com/attachments/942218891952783421/970902343149289514/image.png)
> Detects type of item and the count of the item which is then added to the planner

**This feature could serve as foundation to or be an add on to #469**

**Comment by GalvinGao** [comment]
We've actually planned this before (#441). Thanks for raising this up and this is definitely one of the features we look forward to implement!

**Comment by jonathanlo411** [comment]
Love to see it, looking forward to it as well!

**Comment by FlandiaYingman** [comment]
Yeah @KumoSiunaus and I both had implemented the depot recognizer separately. This might be implemented in the v4 frontend! :D

**Comment by jonathanlo411** [comment]
Good to hear; Any expected timeline on it?

I'd love to help out so reach out if there's anything I can do (though I doubt I could do much lol).

**Comment by GalvinGao** [comment]
> Good to hear; Any expected timeline on it?
> 
> I'd love to help out so reach out if there's anything I can do (though I doubt I could do much lol).

We're currently working on bootstrapping the v3 Frontend development internally. After the kickoff and skeleton-tweaking the repo will be up and I'll keep you updated on that :D
